extensions/mssql/src/hdfs/ui/hdfsManageAccessDialog.ts

```diff
@@ -280,21 +281,21 @@ export class ManageAccessDialog {
 		rowContainer.addItem(nameCell);
 
 		// Access - Read
-		const accessReadComponents = createCheckbox(builder, entry.getPermission(AclEntryScope.access).read, true, permissionsReadColumnWidth, permissionsRowHeight);
+		const accessReadComponents = createCheckbox(builder, entry.getPermission(AclEntryScope.access).read, true, permissionsReadColumnWidth, permissionsRowHeight, `${loc.accessHeader} ${loc.readHeader}`);
```

Comment:
we shouldn't do string concatenation for localizable strings.

Reply:
So for this the reasoning I went with is that this isn't a single sentence - it's two separate words that are used to describe what the box is about. I was also thinking of putting a - inbetween them but wasn't trying to avoid adding more clutter.

In the end I'm fine following the normal way of making a separate string but I felt like this case was special enough to warrant breaking that. 

Thoughts?

Reply:
If it's a simple concat like this where it doesn't chance sentence structure, isn't it okay?  The concern is typically when values are getting inserted mid-sentence.